templates/Web/_composition/Flask/Page.Flask.Grid.AddRoutes/Page.Flask.Grid.AddRoutes/Page.Flask.Grid.AddRoutes/server/constants_postaction.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+CONSTANTS = {}
+
+//{[{
+CONSTANTS['ENDPOINT'] = {}
+//}]}
+//{[{
+CONSTANTS['ENDPOINT']['GRID'] = '/api/grid'
```

Comment:
I am comparing this with the node template. Why is it '/api/grid'? It is only '/grid' in the node template.

link to node: https://github.com/microsoft/WebTemplateStudio/blob/dev/templates/Web/_composition/NodeJS/Page.Node.Grid.AddRoutes/server/constants_postaction.js

Reply:
In Node, '/api' is appended to every route for the routes file. It is essentially, '/api/route' for every route that front-end makes request for.